pysat/tests/test_utils_io.py

```diff
@@ -147,19 +160,88 @@ def test_basic_write_and_read_netcdf_mixed_case_meta_format(self):
 
         self.loaded_inst, meta = io.load_netcdf(
             outfile, pandas_format=self.testInst.pandas_format)
-        keys, new_keys = self.eval_loaded_data()
+        self.eval_loaded_data()
+
+        return
 
-        # Check that names are in the expected case
-        pysat.utils.testing.assert_lists_equal(keys, new_keys)
+    @pytest.mark.skipif(not has_pysat_netcdf,
+                        reason="missing pysat_netcdf instrument module")
```

Comment:
why is this here?  the module is added in this same pull

Reply:
I thought that had been removed.